nicegui/elements/chat_message.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+from ..dependencies import register_component
+from .mixins.disableable_element import DisableableElement
+
+register_component('chat_message', __file__, 'chat_message.js')
+
+
+class ChatMessage(DisableableElement):
+
+    def __init__(self, *,
+                 sent: Optional[bool] = False,
+                 label: Optional[str] = '',
+                 name: Optional[str] = '',
+                 text: Optional[str] = '',
+                 stamp: Optional[str] = None,
+                 avatar: Optional[str] = None,
+                 ) -> None:
+        """Chat Message
+
+        Based on Quasar's `Chat Message <https://quasar.dev/vue-components/chat/`_ component.
+
+        :param sent: Render as a sent message (so from current user) (default: False)
+        :param label: Adding a label adds a tiny headline between messages
+        :param name: The name of the message author
+        :param text: The message body
+        :param stamp: The time stamp of the message. (default: `None`))
+        :param avatar: URL to avatar. (default: `None`))
+        """
+        super().__init__(tag='chat_message')
```

Comment:
I tried that too. Very new into the inner workings of vue, quasar and nicegui. Hopefully one day we will understand. I dont mind you guys taking it from here. I think it is easier than too much ...commenting.

Reply:
It would be very helpful if you could improve the pull-request as much as possible. But if you can not figure out a specific thing like using the "q-chat-message" you can pass the ball to us. No problems. I just wanted to clarify that we would like you to improve the PR as much as possible before handing it over completely.